SPECS/python3/python3.spec

```diff
@@ -99,7 +87,7 @@ Summary:        The libraries and header files needed for Python development.
 Group:          Development/Libraries
 Requires:       expat-devel >= 2.1.0
 Requires:       %{name} = %{version}-%{release}
-Requires:       %{name}-setuptools = %{version}-%{release}
+Requires:       python3-setuptools
```

Comment:
and...this should be an ok circular dependency because this is a runtime requires..and our tooling handles this situation.